src/service/resources-impl.js

```diff
@@ -1721,7 +1729,17 @@ export class Resource {
    * @return {boolean}
    */
   renderOutsideViewport() {
-    return this.element.renderOutsideViewport();
+    const renders = this.element.renderOutsideViewport();
+    // Boolean interface, element is either always allowed or never allowed to
+    // render outside viewport.
+    if (renders === true || renders === false) {
+      return renders;
+    }
+    // Numeric interface, element is allowed to render outside viewport when it
+    // is within X times the viewport height of the current viewport.
+    const viewportBox = this.resources_.getViewport().getRect();
+    const distanceFromViewport = this.layoutBox_.top - viewportBox.bottom;
```

Comment:
This was intentionally modeled after the `amp-ad` `#renderOutsideViewport`: If you're below the element, it should render because you've already passed it. I'm fine changing this to `abs`, though we should probably signal to `Element#renderOutsideViewport` that it's above the user.